docs/management-api-reference.http

```diff
@@ -0,0 +1,153 @@
+#########################################################
+#### Variables
+#########################################################
+
+@subscriptionId = [your-subscription-id]
+@resourceGroup = [your-resource-group-name]
+@armBaseUrl = https://management.azure.com/subscriptions
+@vaultsPath = {{armBaseUrl}}/{{subscriptionId}}/resourceGroups/{{resourceGroup}}/providers/Microsoft.TokenVault/vaults
+@version = ?api-version=2018-08-01-preview
+@vaultName = testVault
```

Comment:
Make this a varable also with [] or else people will try to create with the same name likely

Reply:
Fixed